lib/redis/connection/command_helper.rb

```diff
@@ -24,7 +26,7 @@ def build_command(args)
         command[0] = "*#{(command.length - 1) / 2}"
 
         # Trailing delimiter
-        command << ""
+        command << TRAILING_DELIMITER
```

Comment:
I actually wonder what's the point of appending an empty string...

Reply:
Hmmm...) I don't know) May be remove the line?

Reply:
So I dug the git history and it was added a very long time ago: https://github.com/redis/redis-rb/commit/dcb2498ce58daf276af2b0c7a76db295c842d8b7

So apparently it's so that after you join you have the delimiter at the end:

```ruby
['a', 'b'].join("\n") # => "a\nb"
['a', 'b', ''].join("\n") # => "a\nb\n"
```
So it kinda makes sense.